modules/react/combobox/lib/ComboboxCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {
+  createElemPropsHook,
+  createSubcomponent,
+  ExtractProps,
+} from '@workday/canvas-kit-react/common';
+import {Menu} from '@workday/canvas-kit-react/menu';
+
+import {useComboboxModel} from './useComboboxModel';
+
+export interface ComboboxCardProps<T = any> extends ExtractProps<typeof Menu.Card> {}
+
+export const useComboboxCard = createElemPropsHook(useComboboxModel)(model => {
+  return {
+    minWidth: model.state.width,
+  } as const;
+});
+
+export const ComboboxCard = createSubcomponent('div')({
+  modelHook: useComboboxModel,
+  elemPropsHook: useComboboxCard,
+})<ComboboxCardProps>(({children, ...elemProps}, Element) => {
+  return (
+    <Menu.Card as={Element} {...elemProps}>
```

Comment:
I know the preference here is to use an elemPropsHook, but you could also pass `model` on line 23, and then you can just add `<Menu.Card as={Element} minWidth={model.state.width} {...elemProps}>`. right? Is the elemPropsHook jut cleaner?

Reply:
The preference is to only access the model in elemProps hooks. It is available to the component, but should not be the default.